content/docs/user-guide/project-structure/dvcignore-files.md

```diff
@@ -213,4 +226,4 @@ dir2/data2.dvc:
 ```
 
 Only the second file is displayed because DVC now ignores `data1.dvc` and
-`data1`.
+`dir1`.
```

Comment:
Hmmm this is still confusing. Should be something like "now ignores all files in `dir1/`."